modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -49,42 +45,45 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
-
-After the monitor has run, the list item for the test suite will indicate whether the monitored endpoints passed or failed.
+. Click *Create Monitor* in the *Functional Monitoring* section of Anypoint Monitoring.
+. Under *Step 1 - Setup the monitor*, specify a monitor name. Additionally, toggle *Validate SSL Certificate* to specify if the monitor needs to use self-signed certifications.
+. Tap the *Step 2 - Select endpoints* tab.
+. Populate the fields with the information gathered in "Before you Begin".
++
+You can preview the test by tapping *Run now* under *Preview the result of your monitor*.
 
-[IMPORTANT]
-====
-Tests silently fail when they take more than 120 seconds to run. As a workaround, you can split a long-running test into multiple tests. Download the monitor, split the test across multiple monitors in the BAT CLI, run and time each test to be sure that it is under 120 seconds, and then upload the monitors.
+. Tap the *Step 3 - Set notifications* tab.
+. Optionally configure a notification.
+. Click *Create Monitor*.
 
-If you do not want to split a long-running test, download the monitor and run it from the BAT CLI.
-====
+== Result
 
-== What to do next
+The *Monitor Detail* view includes the following actions:
 
-Click the *Scheduling* tab to see the schedule for the monitor. You can add additional schedules and delete schedules that you no longer want to run. You can also disable schedules, if you want to pause them, and then enable them again.
+* Tap *Run* (image:afm-ui-run-button.png[width=1%,height=1%]) to immediately execute a monitor.
+* Tap *View Monitor History* (image:afm-ui-history-button.png[width=1.3%,height=1.3%]) to view a list of recent monitor runs.
+* Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) followed by *Edit Monitor* to modify a monitor.
+* Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) followed by *Download Monitor* to download a monitor as BAT source.
+* Tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) followed by *Delete Monitor* to delete a monitor.
 
-After the monitor runs, click the *Monitored Endpoints* tab to see:
+Additional displayed information includes:
 
-* Whether all endpoints returned the expected codes, as indicated by *PASSED* in the first column of the table
-* Statistics about the execution of the test of the endpoint
+* The most recent test duration
+* Time elapsed since the monitor started running
+* The average response time for each activity (for example, DNS lookup), separated by day
+* The previous five test execution results, as indicated by *PASSED* or *FAILED* (depending on if the endpoints returned expected codes)
+* A table with the current test schedules
+* Each endpoint asserted with its HTTP method
```

Comment:
Is "asserted" the correct jargon? Just checking....

Reply:
Hey, yup it's a concept that is used throughout the UI.